content/_index.md

```diff
@@ -7,9 +7,41 @@ github_org= "https://github.com/valkey-io"
 github_org_text= "All repos in the Valkey GitHub organization"
 
 headline= "Valkey: an open source, in-memory data store"
-
 long_description= ""
 
+[extra.hero]
+headings = ["FAST.", "RELIABLE.", "OPEN SOURCE, FOREVER."]
+subtitle = "Accelerate your applications with Valkey, the high-performance, in-memory database built for speed, scale, and open innovation. Backed by the Linux Foundation, Valkey delivers the real-time capabilities you need - without vendor lock-in"
```

Comment:
```suggestion
subtitle = "Valkey is an open source (BSD) high-performance key/value datastore that supports a variety of workloads such as caching, message queues, and can act as a primary database. The project is backed by the Linux Foundation, ensuring it will remain open source forever.
```
There was some contention about the wording, so I think we should just revert back to the previous wording to get this merged and someone else can follow up to edit this.